docs/APIs/tasks/wait.md

```diff
@@ -1,6 +1,6 @@
 # Wait
 
-Wait functions provide simple NOP-type wait capabilities. 
+Wait functions provide simple wait capabilities. The OS scheduler will put the current thread in ``waiting state``, allowing another thread to execute. Even better: if there are not other threads in ``ready state``, it can put the whole microcontroller to ``sleep``, saving energy.
```

Comment:
I like it, but it made me notice what is probably a typo in the original - should be "if there are no other threads".